zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -128,6 +128,7 @@
 
     LIST_NOTEBOOK_JOBS,     // [c-s] get notebook job management infomations
     LIST_UPDATE_NOTEBOOK_JOBS, // [c-s] get job management informations for until unixtime
```

Comment:
This is my mistake.
PR is different from the current `Method` requiring unixtime.
`LIST_UPDATE_NOTEBOOK_JOBS` to receive only a portion of the event on the web,
PR of the current transfers that do not require a change in the unixtime server.
Requiring only a limited unixtime operates only API.

therefore,
1. I will fix an erroneous comments on unixtime stamp.
2. For the api to use unixtime stamp will add a comment.

If you have another opinion exists, please tell us.
Thank you.